unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Debugging/Performance/PerformanceMeterController/PerformanceMeterController.cs

```diff
@@ -325,26 +292,26 @@ private void ReportData()
         /// </summary>
         private int CalculatePerformanceScore()
         {
-            double desiredFrameTime = Settings.i.qualitySettings.Data.fpsCap ? 1000/30.0 : 1000/60.0;
-            double frameScore = Mathf.Min((float)(desiredFrameTime/ averageFrameTime), 1); // from 0 to 1
-            double hiccupsScore = 1 - (float) totalHiccupFrames / samples.Count; // from 0 to 1
+            double desiredFrameTime = Settings.i.qualitySettings.Data.fpsCap ? 1000 / 30.0 : 1000 / 60.0;
```

Comment:
Usually to detect stuttering, desiredFrameRate is dynamic and depends on the actual measurements of each machine, but this may fall outside of this PR. What I've seen in the while is the `desiredFrameRate = avg(frames(from p40 to p60))` https://www.vortez.net/articles_pages/frame_time_analysis,2.html

Reply:
extracted it [here](https://github.com/decentraland/unity-renderer/issues/4613)